jenkins/gce-custom-images/prepare_gce_image.sh

```diff
@@ -31,7 +31,6 @@ terra_jupyter_aou_old="us.gcr.io/broad-dsp-gcr-public/terra-jupyter-aou:2.2.13"
 cos_gpu_installer="gcr.io/cos-cloud/cos-gpu-installer:v2.1.9"
 google_cloud_toolbox="us.gcr.io/cos-cloud/toolbox:v20230714"
 docker_composer="docker/compose:1.29.2"
-docker_composer_with_auth="cryptopants/docker-compose-gcr"
```

Comment:
Wait, I think you still need it for GCE (COS) images no? It's dataproc that does not need it

Reply:
oops! yeah I thought I'd gotten rid of all these changes, but I missed one, thanks